noggin/form/edit_user.py

```diff
@@ -80,6 +81,10 @@ def _validate(form, field):
 
 
 class UserSettingsProfileForm(BaseForm):
+    def _https_required(form, field):
```

Comment:
Why make it a member of the class? This construct is a bit unusual, it should either be:
1. outside the class, as before
2. a method, but then it should have `self` as a first argument and called on the instance, which is not what we want for a validator
3. a class method, but then it should be decorated with `@classmethod` and take `cls` as the first argument
4. a static method, but then it should be decorated with `@staticmethod`

I think keeping it outside (option 1) is best here, so it can possibly be reused by another form later.

Reply:
Sorry, I thought it'd be used only for that case, but if it can possibly be reused I agree with option 1. Just did the changes.